content/en/docs/languages/java/_index.md

```diff
@@ -30,6 +30,11 @@ OpenTelemetry Java consists of the following repositories:
   Provides helpful libraries and standalone OpenTelemetry-based utilities that
   don't fit the express scope of the OpenTelemetry Java or Java Instrumentation
   projects. For example, JMX metric gathering.
+- [semantic-conventions-java](https://github.com/open-telemetry/semantic-conventions-java):
+  Auto-generated code for the [OpenTelemetry Semantic Conventions](https://github.com/open-telemetry/semantic-conventions).
+  For example, `ResourceAttributes.SERVICE_NAME`.
+- [opentelemetry-proto-java](https://github.com/open-telemetry/opentelemetry-proto-java):
```

Comment:
This one is questionable [given our weak support](https://github.com/open-telemetry/opentelemetry-proto-java?tab=readme-ov-file#support):

> We have no intention of eventually publishing stable artifacts. If you need guarantees, please generate your own bindings, consulting [grpc codegen](https://grpc.io/docs/languages/java/generated-code/#codegen) and possibly [build.gradle.kts](https://github.com/open-telemetry/opentelemetry-proto-java/blob/main/build.gradle.kts)

I'm still fine including it for completeness, but think we'll see a small uptick in requests for support which we don't intend on providing.